src/FactSystem/ParameterManager.h

```diff
@@ -7,9 +7,7 @@
  *
  ****************************************************************************/
 
-
-#ifndef ParameterManager_H
-#define ParameterManager_H
+#pragma once
```

Comment:
yay!

Reply:
Yeah, if you're even in a file working and see the old style switch it over.